airbyte-integrations/connectors/destination-mysql/src/testFixtures/kotlin/io/airbyte/integrations/destination/mysql/MysqlContainerFactory.kt

```diff
@@ -11,6 +11,6 @@ import org.testcontainers.utility.DockerImageName
 /** Much like the destination-postgres PostgresTestDatabase, this was copied from source-mysql. */
 class MySQLContainerFactory : ContainerFactory<MySQLContainer<*>>() {
     override fun createNewContainer(imageName: DockerImageName?): MySQLContainer<*> {
-        return MySQLContainer(imageName?.asCompatibleSubstituteFor("mysql"))
+        return MySQLContainer(imageName?.asCompatibleSubstituteFor("io/airbyte/integrations/destination/mysql"))
```

Comment:
what is this? afaict `mysql` is meant to be a docker image name, not a java package name?

Reply:
so stupid refactoring coming from me moving files into the kotlin directory...